index.js

```diff
@@ -1,95 +1,226 @@
-  profileIcon.classList.remove("navigation-item--active");
+  removeHighlighting(homeIcon);
+  addHighlighting(bookmarkIcon);
+  removeHighlighting(createCardIcon);
+  removeHighlighting(profileIcon);
 
   heading.textContent = "Bookmarks";
 });
 
 createButton.addEventListener("click", () => {
-  homeSection.classList.add("hidden");
-  bookmarkSection.classList.add("hidden");
-  createSection.classList.remove("hidden");
-  profileSection.classList.add("hidden");
+  hideSection(homeSection);
+  hideSection(bookmarkSection);
+  showSection(createSection);
+  hideSection(profileSection);
 
-  homeIcon.classList.remove("navigation-item--active");
-  bookmarkIcon.classList.remove("navigation-item--active");
-  createIcon.classList.add("navigation-item--active");
-  profileIcon.classList.remove("navigation-item--active");
+  removeHighlighting(homeIcon);
+  removeHighlighting(bookmarkIcon);
+  addHighlighting(createCardIcon);
+  removeHighlighting(profileIcon);
 
-  heading.textContent = "Create Question";
+  heading.textContent = "Create Card";
 });
 
 profileButton.addEventListener("click", () => {
-  homeSection.classList.add("hidden");
-  bookmarkSection.classList.add("hidden");
-  createSection.classList.add("hidden");
-  profileSection.classList.remove("hidden");
+  hideSection(homeSection);
+  hideSection(bookmarkSection);
+  hideSection(createSection);
+  showSection(profileSection);
 
-  homeIcon.classList.remove("navigation-item--active");
-  bookmarkIcon.classList.remove("navigation-item--active");
-  createIcon.classList.remove("navigation-item--active");
-  profileIcon.classList.add("navigation-item--active");
+  removeHighlighting(homeIcon);
+  removeHighlighting(bookmarkIcon);
+  removeHighlighting(createCardIcon);
+  addHighlighting(profileIcon);
 
   heading.textContent = "Profile";
 });
 
-/* SET NAVIGATION ACTIVE BUTTON */
+function showSection(section) {
+  section.classList.remove("hidden");
+}
+
+function hideSection(section) {
+  section.classList.add("hidden");
+}
+
+function addHighlighting(icon) {
+  icon.classList.add("navigation-item--active");
+}
+
+function removeHighlighting(icon) {
+  icon.classList.remove("navigation-item--active");
+}
+
+/* CREATE CARD FORM */
+const createCardForm = getElement("createCardForm");
+/* PREVENT DEFAULT*/
+createCardForm.addEventListener("submit", (event) => event.preventDefault());
+
+// CREATE HTML FOR CARD
+let cardContainer = createHTML();
+let cardContainerChildren = cardContainer.children;
+console.log(cardContainerChildren);
+
+// TAKE INPUT VALUES OF FORM
+createCardForm.addEventListener("submit", () => {
+  const createCardQuestion = getElement("createCardQuestion");
+  const createCardAnswer = getElement("createCardAnswer");
+
+  let questionTextInput = createCardQuestion.value;
+  let answerTextInput = createCardAnswer.value;
+
+  // LOOP OVER NODELIST TO REACH HTML ELEMENTS ON NEW CARD
+  for (let i = 0; i < cardContainerChildren.length; i++) {
```

Comment:
Even though this is a valid way to loop through those children, it is a very `imperative` way. By using `cardContainerChildren.forEach`, you would go with a more `declarative` approach, which is a bit more readable and less error prone.